packages/teleport-plugin-react-app-routing/src/index.ts

```diff
@@ -31,8 +37,10 @@ export const createPlugin: ComponentPluginFactory<AppRoutingComponentConfig> = (
 
   const reactAppRoutingComponentPlugin: ComponentPlugin = async (structure) => {
     const { uidl, dependencies, options } = structure
+    // @ts-ignore-next-line
+    const { useFolderStructure, disableDOMInjection, flavour } = options.meta || {}
```

Comment:
I think you're not using disableDOMInjection anymore right?

Reply:
Yes, I removed it. I missed it here, will remove 👍